docs/Project.toml

```diff
@@ -8,5 +8,5 @@ NLPModels = "a4795742-8479-5a88-8948-cc11e1c8c1a6"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 
 [compat]
-Documenter = "~0.21"
+Documenter = "~0.24"
```

Comment:
You need to move `assets` inside `Documenter.HTML` (docs/make.jl) with Documenter 0.24.

Reply:
What will it change? The build succeeded.

Reply:
Done here: #36

Reply:
It sends a warning during the build, but I'm not sure it doesn't appear to have failed yet: https://juliasmoothoptimizers.github.io/NLPModelsIpopt.jl/latest/

Reply:
Ah, but master hasn't finished building yet, so it may turn out to break the docs. Waiting: https://travis-ci.org/JuliaSmoothOptimizers/NLPModelsIpopt.jl/builds/651763277?utm_medium=notification&utm_source=github_status

Reply:
Dominique: `assets` are not take into account (no more JSO logo :/).

This package doesn't build doc now, like NLPModel (-> view config)